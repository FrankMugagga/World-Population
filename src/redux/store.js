import { configureStore } from '@reduxjs/toolkit';
import countriesReducer from '../features/countries/countriesSlice';

const store = configureStore({
  reducer: {
    countries: countriesReducer,

  },

});

export default store;
