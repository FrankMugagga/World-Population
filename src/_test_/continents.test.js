import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom';
import Continents from '../features/countries/continents';
import store from '../redux/store';

describe('performing test for rockets', () => {
  it('testing if rockets page really matches', () => {
    const rocketRender = renderer
      .create(
        <Provider store={store}>
          <Continents />
        </Provider>,
      )
      .toJSON();
    expect(rocketRender).toMatchSnapshot();
  });
});
