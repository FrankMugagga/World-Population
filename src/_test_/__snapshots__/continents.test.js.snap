// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performing test for rockets testing if rockets page really matches 1`] = `
<div>
  Loading...
</div>
`;
